server/models/Room.js

```diff
@@ -0,0 +1,40 @@
+const { Schema, model, trusted } = require('mongoose');
+
+const roomSchema = new Schema(
+  {
+    name: {
+      type: String,
+      required: true,
+      trim: trusted,
```

Comment:
I believe this is meant to be `trim: true` and not `trusted`.  If so, this can be changed and trusted can be removed from the imports

Reply:
-Changed Trim to be 'true' instead of 'trusted'